podpac/core/coordinates/coordinates.py

```diff
@@ -692,6 +733,19 @@ def shape(self):
 
         return tuple(size for c in self._coords.values() for size in c.shape)
 
+    @property
+    def ushape(self):
+        if len(self.dims) == len(self.udims):  # shortcut in case no stacking
+            return self.shape
+
+        ushape = []
+        for ud in self.udims:
+            for i, d in enumerate(self.dims):
+                if ud in d:
+                    ushape.append(self.shape[i])
+                    break
+        return tuple(ushape)

```

Comment:
I think you can just do

```
@property
def ushape(self):
    return tuple(self[dim].size for dim in self.udims)
```

Reply:
AH! I was looking for something like that! THanks!